src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -815,6 +795,21 @@ struct CodeGen
         return array;
     }
 
+    llvm::Value* loadClassObjectFromPool(PoolIndex<ClassInfo> index)
```

Comment:
Does this no longer need `generateEHDispatch`?

Reply:
As long as we don't verify class files not really no. Previously it did so since we also used to run the class initializer, but since that is no longer the case it won't throw any exceptions anymore